Database/Corrections/Wotlk/wotlkQuestFixes.lua

```diff
@@ -1385,9 +1430,6 @@ function QuestieWotlkQuestFixes:Load()
         [13122] = {
             [questKeys.preQuestSingle] = {13104},
         },
-        [13136] = {
-            [questKeys.objectives] = {nil,nil,{{43259}}},
```

Comment:
Why are you removing this objective fix from https://github.com/Questie/Questie/commit/be1c20e078fa9cf1ad62ee0539594b6c39b26385 ?

Reply:
This was actually moved a bit higher (the order of the corrections is a bit off). So the corrections stays 👍🏻

Reply:
The problem with this quest is that it can drop from only one mob type, but it's not a "unique location" mob, so you end up with a bunch of ! all over your map. In testing that's why I removed it.

Trying to avoid situations where we have a bunch of extra !... like the ticket mentions.

https://www.wowhead.com/wotlk/npc=30597/spiked-ghoul

Reply:
Hmm that explains why you removed that other quest further down, but you added this quest (Jagged Shard one) back in again a bit higher in the file. Probably an oversight?